db/migrate/20220125014850_add_state_to_bulk_tax_return_updates.rb

```diff
@@ -0,0 +1,6 @@
+class AddStateToBulkTaxReturnUpdates < ActiveRecord::Migration[6.1]
+  def change
+    add_column :bulk_tax_return_updates, :state, :string
+    add_index :bulk_tax_return_updates, :state
```

Comment:
do we look up these records by state a lot? just wondering what the reason behind the index is